src/js/App/RootApp.js

```diff
@@ -53,7 +53,7 @@ RootApp.propTypes = {
   globalFilterHidden: PropTypes.bool,
 };
 
-function stateToProps({ chrome: { activeApp, activeLocation, appId, pageAction, pageObjectId, globalFilterHidden } }) {
+function stateToProps({ chrome: { activeApp, activeLocation, appId, pageAction, pageObjectId }, globalFilter: { globalFilterHidden } }) {
```

Comment:
This killed tests. We will probably need some default value.

Reply:
Good point!